lib/rubocop/cop/mixin/range_help.rb

```diff
@@ -51,9 +51,13 @@ def range_with_surrounding_comma(range, side = :both)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
-      def range_with_surrounding_space(range:, side: :both,
-                                       newlines: true, whitespace: false,
-                                       continuations: false)
+      NOT_GIVEN = Module.new
```

Comment:
Just curious – why is `NOT_GIVEN` a module and not e.g. a symbol?

Reply:
`Module.new == Module.new` evaluates to `false`. Chances of someone passing a `:not_given` positional argument are nearing zero, but anyway.

Reply:
Got it, thanks.